Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 # Temporarily limiting rake version:
 # #http://stackoverflow.com/questions/35893584/nomethoderror-undefined-method-last-comment-after-upgrading-to-rake-11
-gem 'rake', '~> 10.0'
```

Comment:
Let's continue to use pessimistic version constraints unless there's a reason to lock a specific version (and if we're locking the version, we should be sure to leave a comment listing the reason).